sdk/eventhub/azure-eventhub/CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Release History
 
```

Comment:
do we need to add the Python 3.7+ comment or no?
`This version and all future versions will require Python 3.7+. Python 2.7 and 3.6 are no longer supported.`